.travis.yml

```diff
@@ -116,21 +120,38 @@ matrix:
         # location during testing. We also use this build to make sure that the
         # dependencies get correctly installed with pip.
         - os: linux
+          arch: amd64
```

Comment:
I suppose even these are unnecessary as the amd64 is the default (the not sure part is it might not stay at `arm64` for the jobs after it's changed

Reply:
yeah, if the build works and passes tests I'll be astonished and then start figuring out the actual right way to do all this

Reply:
It would probably be more sensible to do this on an actual arm64 at home but my Raspberry Pi 4 isn't here.

Reply:
yes, I can confirm that these are unnecessary. I've also tried, but sadly the arm jobs still only just support C as language, so currently no way around of installing python.